### PR TITLE
[full-ci] Prepare Release 4.0.0

### PR DESCRIPTION
**Pull request by kobergj**
prepares 4.0.0 release

### CODE REVIEW

File: ocis-pkg/version/version.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ var (
 	// LatestTag is the latest released version plus the dev meta version.
 	// Will be overwritten by the release pipeline
 	// Needs a manual change for every tagged release
-	LatestTag = "3.1.0-rc.1+dev"
+	LatestTag = "4.0.0"
```
Comment (micbar) [inline ocis-pkg/version/version.go:19]:
```suggestion
	LatestTag = "4.0.0+dev"
```

Comment (micbar) [reply ocis-pkg/version/version.go]: The final release tag is overwritten by the build. Please always keep the `+dev` meta version. It needs to be there for local dev builds. All CI builds overwrite the LatestTag with the real git version.

**Review by micbar — COMMENTED** [review]
Some nitpicks

**Comment by micbar** [comment]
@kobergj @JammingBen @JanAckermann is the web version already the final 7.1.0? Or do we need another bump?

**Comment by kobergj** [comment]
There was no web bump mentioned in the release next step (https://github.com/owncloud/ocis/issues/6838#issuecomment-1675000741) . If we need one then the release will probably be delayed.

**Comment by micbar** [comment]
That is true. That was overlooked or already done.

**Comment by JammingBen** [comment]
Web still needs the final 7.1.0 bump, I was waiting for the final go. I briefed @fschade so he can do releases during my absence.

**Comment by micbar** [comment]
@kobergj @fschade that gives us two options:

1) if ocis master has no new changes, we could bump on master and rebase the release branch
2) bump web on the release branch

**Comment by kobergj** [comment]
We anyway need to rebase to master because of deprecations and other fixes. So the web bump can target master directly

**Review by butonic — APPROVED** [review]

**Pull request merged**